Source/Core/Core/Boot/Boot.cpp

```diff
@@ -244,6 +244,17 @@ bool CBoot::DVDRead(const DiscIO::VolumeDisc& disc, u64 dvd_offset, u32 output_a
   return true;
 }
 
+bool CBoot::DVDReadDiscID(const DiscIO::VolumeDisc& disc, u32 output_address)
+{
+  std::array<u8, 0x20> buffer;
+  if (!disc.Read(0, 0x20, buffer.data(), DiscIO::PARTITION_NONE))
+    return false;
+  Memory::CopyToEmu(output_address, buffer.data(), 0x20);
```

Comment:
Simply laziness &mdash; ideally, `DVDReadDiscID` wouldn't exist at all and it would instead just directly call code in DVDInterface (which would also manage the drive state change), but that doesn't work well with DVDInterface being async.  I'll change it to `buffer.size()` for now though.